crates/bindings-sys/src/lib.rs

```diff
@@ -61,7 +61,11 @@ pub mod raw {
         ///
         /// The table id is written into the `out` pointer.
         ///
-        /// Returns an error if the table does not exist.
+        /// Returns an error if
+        /// - a table with the provided `table_id` doesn't exist
+        /// - the slice `(name, name_len)` is not valid UTF-8
+        /// - `name + name_len` overflows a 64-bit address.
```

Comment:
`name_len` computations: I thought this was all in wasm32, and so usize is 32 bits?